bokehjs/src/coffee/core/selection_manager.coffee

```diff
@@ -24,7 +24,7 @@ export class SelectionManager extends HasProps
 
   inspect: (renderer_view, geometry) ->
     did_hit = false
-    @_set_inspector(renderer_view)
+    @_set_inspector(renderer_view.model)
```

Comment:
This is an annoying side-effect to do. The inspector should probably be created in the renderer.initialize method.